src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckClassVisitor.kt

```diff
@@ -89,7 +90,11 @@ internal class LincheckClassVisitor(
                 mv
             }
         }
-        if (methodName == "<clinit>") {
+        if (methodName == "<clinit>" ||
+            // Debugger implicitly evaluates toString for variables rendering
+            // We need to disable breakpoints in such a case, as the numeration will break
```

Comment:
How does it help to disable breakpoints? Please clarify.

Reply:
We do not instrument toString and enter ignored section, so there are no beforeEvents inside

Reply:
Please clarify in the documentation